datalad/distribution/create_sibling_github.py

```diff
@@ -165,17 +169,29 @@ def __call__(
             access_protocol='https',
             publish_depends=None,
             private=False,
-            dryrun=False):
+            dryrun=False,
+            dry_run=False):
```

Comment:
Don't these need to be merged internally?  (It looks like the new `dry_run` isn't considered downstream.)

Also, I think it'd be good to add a `warnings.warn(..., DeprecationWarning)`.

Reply:
:facepalm:

Reply:
Done now, thx for pointing it out!